### ISSUE TITLE
2.0 - Android - Tab content not visible when top tab navigator is nested in bottom navigator and swipe/ animation is enabled.

### ISSUE THREAD

**Issue by llamaluvr** [react-navigation/react-navigation#4071]
### Current Behavior

![screenshot_20180426-120618](https://user-images.githubusercontent.com/8053974/39317879-b00f39d4-494a-11e8-977c-a2f6c02d5fcd.png)

In 2.0.0-rc.1 (and 2.0.0 latest from master), when a top tab bar is nested inside a bottom tab bar, and animationEnabled and swipingEnabled is true on the top tab bar and false on the bottom tab bar, no content appears in the top tabs and the top tab indicator never locks to a particular tab, instead tracking your finger.

In 1.5.11, in the same scenario, content appears in the top tabs and swiping between them works correctly.

The following snack demonstrates the issue (well, sorta): https://snack.expo.io/@llamaluvr/bug:-top-tab-blank-when-swipe-is-enabled

I saw "sorta" because, in the Snack web UI, I can't for the life of me get React Navigation 2.0.0  to work. I get "undefined is not a function (evaluating '(0, _reactNavigation.createMaterialTopTabNavigator)'. It works just fine if I switch it back to 1.5.11 (which installs by default when you include react-navigation as an import in Snack, so maybe it's a snack caching issue?). If I download this Snack to the XDE on my desktop, I can reproduce the issue using React Navigation 2.0.0).

### Expected Behavior

I would expect 2.0.0 to work the same as 1.5.11. Content should appear in the top tabs and you should be able to swipe between them, as long as swiping isn't enabled in the bottom tab navigator.

### How to reproduce

1. Create a material top tab navigator. Put 2+ tabs inside. Set animationEnabled and swipeEnabled to true.
2. Create a bottom tab navigator. Set the top tab navigator as one of the routes. Set animationEnabled and swipeEnabled to false (failing to do this would cause the same bug to appear on react-navigation 1.5.11).
3. Run the app.
4. You'll see that a) the tab content does not show in the top tabs, and b) that when you swipe, the tab indicator never locks into position, instead sliding with your finger like you're playing Super Breakout with it.

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 2.0.0-rc.1 / 2.0.0 (latest from master)
| react-native     | 54 / Expo 26
| node             | 8.4
| npm or yarn      | yarn 1.6

**Comment by brentvatne** [comment]
The example you provided is invalid, if you ran it locally (with dev=true) you'd see that it attempts to render multiple independent navigators.

```
const BottomTabs = createBottomTabNavigator(
  {
    BottomTab1: () => <TopTabs />,  // <------------- here
    BottomTab2: () => <TabScreen title="Bottom tab2" />,
  },
```

if you change that and then re-arrange so that BottomTabs is below TopTabs declaration, it works as expected for me. gist: https://gist.github.com/brentvatne/8704adcc9053f100e7d434e3360e49d9

**Comment by llamaluvr** [comment]
I did have problems with that code originally, and then I changed it. Apologies for that - I'm not sure why my Snack was on that old version of my code. I just tried updating the Snack again, but either way here is the actual code I last tried:

```
import React, { Component } from 'react';
import { Text, View, StyleSheet } from 'react-native';
import { /*TabNavigator,*/ createMaterialTopTabNavigator, createBottomTabNavigator } from 'react-navigation';
import { Constants } from 'expo';

//const createMaterialTopTabNavigator = TabNavigator;
//const createBottomTabNavigator = TabNavigator;
 
class TabScreen extends Component {
  render () {
    return (
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center'}}><Text>{this.props.title}</Text></View>
    );
  }
}

class TopTab1 extends Component {
  render () {
    return (<TabScreen title="Top tab1" />);
  }
}

class TopTab2 extends Component {
  render () {
    return (<TabScreen title="Top tab2" />);
  }
}

const TopTabs = createMaterialTopTabNavigator(
  {
    'TopTab1': TopTab1,
    'TopTab2': TopTab2,
  },
  {
    initialRouteName: 'TopTab2',
    tabBarPosition: 'top',
    swipeEnabled: true,
    animationEnabled: true,
    tabBarOptions: {
      style: {
        paddingTop: Constants.statusBarHeight,
      } 
     }
  }
);

class BottomTab2 extends Component {
  render () {
    return (<TabScreen title="Top tab2" />);
  }
}

const BottomTabs = createBottomTabNavigator(
  {
    'BottomTab1': TopTabs,
    'BottomTab2': BottomTab2,
  },
  {
    tabBarPosition: 'bottom',
    swipeEnabled: false,
    animationEnabled: false,
  }
);

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <BottomTabs />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
});
```

I did try your gist in my XDE, and got the same behavior as the code pasted above - works great in Expo 27, doesn't work in Expo 26. So hopefully all is good when I upgrade my actual project (crossing my fingers since I also have a switch and a stack involved). Thanks for looking!

**Comment by dbasedow** [comment]
I had the same issue. This seems to be fixed in react-native 0.55.0
